WordPress/src/test/java/org/wordpress/android/ui/sitecreation/sitename/SiteCreationSiteNameViewModelTest.kt

```diff
@@ -0,0 +1,55 @@
+package org.wordpress.android.ui.sitecreation.sitename
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.nhaarman.mockitokotlin2.verify
+import kotlinx.coroutines.CoroutineDispatcher
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.ui.sitecreation.misc.SiteCreationTracker
+import org.wordpress.android.util.LocaleManagerWrapper
+
+@RunWith(MockitoJUnitRunner::class)
+class SiteCreationSiteNameViewModelTest {
+    @Rule
+    @JvmField val rule = InstantTaskExecutorRule()
+
+    @Mock lateinit var analyticsTracker: SiteCreationTracker
+    @Mock lateinit var localeManagerWrapper: LocaleManagerWrapper
```

Comment:
I guess this isn't needed here.

Reply:
Cleaned up with https://github.com/wordpress-mobile/WordPress-Android/pull/16335/commits/b2379edd2c316246046c6f49109d21f5de54083f.